README.rst

```diff
@@ -0,0 +1,42 @@
+============
+magic-marker
+============
+
+
+.. image:: https://img.shields.io/travis/rcbops/magic-marker.svg
+        :target: https://travis-ci.org/rcbops/magic-marker
+
+
+Parse JUnitXML and upload test results to qTest Manager.
+
+Quick Start Guide
+-----------------
+
+1. Install ``magic-marker`` from PyPI with pip::
+
+    $ pip install magic-marker
+
+2. For more information on using the magic-marker launch help by::
+
+    $ magic-marker --help
```

Comment:
Incorrect, the executable name is `magic_marker` according to the "setup.py" file.

Reply:
changed in setup.py